h2o-py/demos/xgboost_feature_interactions_demo.ipynb

```diff
@@ -0,0 +1,1126 @@
88suxOff7mvvWtb2Xzl19+OezcfPPNZf+exYsXh7MBAwZk8xEjRoSdaH3OCP23pn5eo+c0pfh5LbW26HmNHmepn9dYf78pVfaxAtvmwx/+cDj729/+1ogrqRzfOAwAAAAAAAAAAAAABeDgMAAAAAAAAAAAAAAUgIPDAAAAAAAAAAAAAFAADg4DAAAAAAAAAAAAQAE4OAwAAAAAAAAAAAAABVDb1AtoTLvvvnvZnZdffjmc7bffftuyHKCRfOhDH8rmr776atgpNfvoRz+6zWv6IP7yl79k81122SXsDB48OJt36tQp7MydO7e8haWUZsyYUXanuVq3bl02L/X+0BzV1NSEs0MOOSScHXjggdm8VatWYefFF1/8wOvaFi+88ELZnf33378BVtK8zJs3L5wNHTq0EVfyf9tuu/j/efv2t79dVv5+NmzYkM332GOPsHPiiSdm81Kv20W5hn72s5+Fnauvvrrs39NYevbsWfZs5cqVDbWcf1HN12p9/eQnP8nmpR7PEUcc0VDL+RdFuVZLaazX4Oj1N6X4NTh6/U2p9Gsw0DBeeumlcFbq3gLqK9rz3bx5c9iJ7vVTSqlLly7bvCag6UR7wSnFe77VvBecUvyZNtoLTineD7YXXFqp94ei7AfbC65+0f15qXtw9+fVsZf2xBNPhLPmuEdaau+0sfZIK60+rwv16ZR6v6mkjh07ZvPtt98+7Oy0005l/57Zs2dn8zPPPLPsn1XKJz7xiXB22WWXZfMRI0aEnejvtG/fvuUtrIVq6uc1ek5Tip/XUq890fMaPc6UKvtY6/P3m1JlHyuwbUrdc3zuc59rxJVUjm8cBgAAAAAAAAAAAIACcHAYAAAAAAAAAAAAAArAwWEAAAAAAAAAAAAAKAAHhwEAAAAAAAAAAACgABwcBgAAAAAAAAAAAIACcHAYAAAAAAAAAAAAAAqgtqkX0Jh69OgRznbaaadsfvfdd4ed733ve9u8JqDhfepTn8rmdXV1YWf8+PHh7Pbbb9/mNf2PNWvWhLOZM2dm80GDBoWdBx54IJvPmDGjvIW9jy1btmTzUv+m9VHpn1cfs2bNyubnnHNOI6+kYbVt2zacbdy4MZx16dIlm5966qlh59FHH83mEyZMCDv1Ef2e1q1bh52TTjqpYr//vffeC2fbbdf0/+/Whg0bsvnChQvDzqRJkxpqOVXn7LPPzualXpeuuuqqsn9PUa6hcePGVWRN2+Lcc88NZ7fccks2X7duXQOt5oNraddq9L6aUkrt2rXL5kOHDm2o5fyLRYsWhbP+/ftn82q+VlOKr9dKvt9VWvT6m1L8Glyf119g2/3+97/P5s8991zYOfLIIxtoNRRZ7969s3n79u3Dzrx588LZWWedtc1rAppOtBecUvx5spr3glOK94OjveCUKrsfHO0Fp1TZ/dtq3gtOqTj7wY21F5xSZe+Pq+HeONoProa94Pporvfn1byXtnTp0kb5PaVEe6TR/mhK1bFHWkmlXi9OP/30sn9e9NpY6v432oe84IILyv79pbzxxhvZvNR7+6GHHlr279l1112z+T/+8Y+yf1Z9HXTQQWV3OnXq1AAraTma+nltrOc0epwptbzHCnwwL730UjZfvnx52Jk4cWJDLadBNc87FQAAAAAAAAAAAACgLA4OAwAAAAAAAAAAAEABODgMAAAAAAAAAAAAAAXg4DAAAAAAAAAAAAAAFICDwwAAAAAAAAAAAABQALVNvYDGVFsbP9zTTz89m1999dVhZ+zYsdm8Q4cO5S0MCurtt98uu/POO++U3RkwYEA2P/jgg8POrFmzwtmmTZuy+bBhw8JO9FjnzZsXdmbPnp3N16xZE3Yit956azg75JBDsvmyZcvCzm9/+9ts/tprr4Wd1atXZ/POnTuHnXXr1oWzyIYNG8rulPLd7363rJz/du6554az3r17Z/Pf/OY3Yadnz57Z/J///GfY+fGPf5zNJ02aFHZK/T1GfvSjH2Xziy++OOz8+te/DmcHHHBA2Wuoj4ULF2bzXXbZJez06tWroZbTJH7wgx+Es7lz52bzX/3qV2Fnjz322OY1/Y8iXUOU1lyv1ehaufbaa8POaaedls1vvPHGsn9/XV1dOIs+k3ziE58IO/3798/mjXWtphRfr9G1mlJ8vVbDtRq9BkevvynFf1eVfP0FPrjo8+5+++0Xdvr27dtQy6HA2rVrl82jzxYppXT55ZeHsxEjRmRze76wbZp6LzileD+4mveCU6rsfnC0F5xSvB8c7QWnFO8HR/ddKcX3I9W8F/x+s6Krz71xSvH9cTXfG0d7wSnFn4+rYS+4lJZ2f95c99Jamsceeyybf+973ws7o0ePzuZf/epXw06bNm2y+fz588NOdP+QUkrDhw8PZ+WaOnVqOOvTp082L/XfHwYPHlz2Gm6//fZs3qNHj7BT6j4qMmbMmGxe6r6r1H9n+MhHPlL2Gh544IFsfvjhh4edUvsXlVSfz7Rbt25tgJWUp5LPayWf05Ti57U+z2n0OFOKH2tj/f2mVNnHCnwwkydPzuZ77bVX2Dn66KMbaDUNyzcOAwAAAAAAAAAAAEABODgMAAAAAAAAAAAAAAXg4DAAAAAAAAAAAAAAFICDwwAAAAAAAAAAAABQAA4OAwAAAAAAAAAAAEABODgMAAAAAAAAAAAAAAVQ29QLqBbjx4/P5j/72c/Czje/+c1sfvPNN1dkTdASPPXUU+Hs4osvLvvn3Xrrrdm8e/fuYWfw4MHZ/P777w87Z599djh78MEHs/ljjz0Wdo455phsftttt4WdXXfdtaw8pZTOPPPMbH777beHnTFjxmTzKVOmhJ2pU6dm86997WthJ3q+S/1bX3/99eEscscdd4SzT37yk9n82GOPDTs77rhj2WsgpQMPPDCcPfLII9n88ssvDzvdunXL5i+++GLYOeOMM7L5uHHjwk591NbmP061b98+7Gy3XdP/v1vz5s3L5ieccEIjr6QyVq5cmc0nT5
```

Comment:
I think `accuracy` is not super precise in this context. Gain is a rather a measure of loss reduction.